### PR TITLE
Add entry points to the windows docker files

### PR DESCRIPTION
**Pull request by TravisEz13**
Related to #2793

### CODE REVIEW

File: docker/release/nanoserver/Dockerfile [hunk 1]
```diff
@@ -46,3 +46,5 @@ RUN Start-Transcript -path C:\Dockerfile.log -append -IncludeInvocationHeader ;
 
 # Persist %PSCORE% ENV variable for user convenience
 ENV PSCORE='"C:\Program Files\PowerShell\Core\PowerShell.exe"'
+
+ENTRYPOINT ["cmd", "/c", "\"%PSCORE%\""]
```
Comment (SteveL-MSFT) [inline docker/release/nanoserver/Dockerfile]: You can't just call powershell.exe directly?

Comment (TravisEz13) [inline docker/release/nanoserver/Dockerfile]: done

File: docker/release/nanoserver/Dockerfile [hunk 2]
```diff
@@ -46,3 +46,5 @@ RUN Start-Transcript -path C:\Dockerfile.log -append -IncludeInvocationHeader ;
 
 # Persist %PSCORE% ENV variable for user convenience
 ENV PSCORE='"C:\Program Files\PowerShell\Core\PowerShell.exe"'
+
+ENTRYPOINT ["C:\\Program Files\\PowerShell\\Core\\PowerShell.exe"]
```
Comment (SteveL-MSFT) [inline docker/release/nanoserver/Dockerfile]: Typically, powershell.exe is expected to be installed to a versioned folder (`c:\program files\powershell\6.0.0-alpha.17\powershell.exe`), how does `Core` fit into this?

Comment (TravisEz13) [inline docker/release/nanoserver/Dockerfile]: The docker image creates the core folder as a symbolic link (see line 35).  This allow this static path to be used.

Comment (SteveL-MSFT) [inline docker/release/nanoserver/Dockerfile]: Since it's just a symlink, why not just put it in `C:\Program Files\PowerShell`?

Comment (TravisEz13) [inline docker/release/nanoserver/Dockerfile]: Done

File: docker/release/windowsservercore/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # escape=`
-FROM microsoft/windowsservercore
+FROM microsoft/windowsservercore:latest
```
Comment (SteveL-MSFT) [inline docker/release/windowsservercore/Dockerfile]: How come you don't have this suffix for nanoserver?

Comment (TravisEz13) [inline docker/release/windowsservercore/Dockerfile]: I didn't mean to add it, but it should be there.  I'll add it too.

File: docker/release/nanoserver/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # escape=`
-FROM microsoft/nanoserver
+FROM microsoft/nanoserver:latest
 MAINTAINER brycem@microsoft.com
```
Comment (SteveL-MSFT) [inline docker/release/nanoserver/Dockerfile]: Should we change this to powershellteam@hotmail.com?

Comment (TravisEz13) [inline docker/release/nanoserver/Dockerfile]: Does that go anywhere?  All the rest have actual email addresses on them.  If we want to update them, that should probably be a separate item.  The MAINTAINER tag is deprecated anyway.  I filed #3465 for this.  Can we continue this discussion there?

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**